package.json

```diff
@@ -17,7 +17,7 @@
     "generate": "stencil generate"
   },
   "dependencies": {
-    "@ionic/core": "^7.5.0"
+    "@ionic/core": "7.6.2-dev.11704729332.13b412c8"
```

Comment:
The dev build is needed in order to import the dark theme files. This will be reverted once the feature is released. Jira ticket has been created.